addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -16,31 +16,42 @@ Note that the commands are executed in the context and with the privileges of th
 
 The `command` Thing requires the command to execute on the shell. Optionally one can specify:
 
-- a transformation to apply on the execution result, 
-- an interval, in seconds, the command will be repeatedly executed, 
-- a time-out, in seconds, the execution of the command will time out, and lastly, 
-- a boolean parameter to make the command execute immediately every time the state of the input channel has changed. 
+- [transform] a transformation to apply on the execution result, 
+- [interval] an interval, in seconds, the command will be repeatedly executed, 
+- [timeput] a time-out, in seconds, the execution of the command will time out, and lastly, 
+- [runOnInput] a boolean parameter to make the command execute immediately every time the state of an input channel is set by a Command 
+- [repeatEnabled] a boolean parameter to allow the command execute repeatedly, e.g. when the state of an input channel is set repeatedly with the same state. This parameter only makes sense when used in combination with [runOnInput] set to true.
```

Comment:
Again, the item state is not what is driving the execution. This should be described as ignoring repeated _commands_ to the input channel. Just curious, is there a specific use case (or cases) you have in mind for this configuration? If there is a debounce-type scenario that couldn't that be handled by repeatedly setting the input _state_ and using rules to throttle the execution via the run channel. This approach supports more complex throttling, such as n commands per time interval (but must be implemented by the user).